src/init.cpp

```diff
@@ -745,17 +745,17 @@ static void ThreadImport(std::vector<fs::path> vImportFiles)
  */
 static bool InitSanityCheck()
 {
-    if(!ECC_InitSanityCheck()) {
-        InitError("Elliptic curve cryptography sanity check failure. Aborting.");
-        return false;
+    if (!ECC_InitSanityCheck()) {
+        std::string rare_error_do_not_translate = "Elliptic curve cryptography sanity check failure. Aborting.";
+        return InitError(bilingual_str{rare_error_do_not_translate, rare_error_do_not_translate});
```

Comment:
e985a077f5f6b5e154f2cb4be25a50a45e00e211

These changes are unnecessary if you keep `error InitError(const std::string&)` and just do:
```cpp
bool InitError(const std::string& str)
{
    return InitError(bilingual_str{str, str});
}
```

Reply:
The function with signature `InitError(const std::string&)` has been removed intentionally to prevent emerging of untranslated error messages in the future. All such cases should be implemented in the code explicitly.